app/enterprise/2.1.x/deployment/installation/amazon-linux-2.md

```diff
@@ -125,7 +125,7 @@ You should now have two files in your home directory on the target Amazon system
 Execute a command similar to the following, using the appropriate RPM file name you downloaded.
 
 ```bash
-$ sudo yum install kong-enterprise-edition-{{page.kong_latest.version}}.aws.rpm
+$ sudo yum install kong-enterprise-edition-{{page.kong_latest.version}}.*.rpm
```

Comment:
hm, good question: we have:

- `kong-enterprise-edition-amzn1.<arch>.rpm`
- `kong-enterprise-edition-amzn2.<arch>.rpm`

where `arch` can be: `amd64` and `noarch` for EE